src/components/general/Icon/IShare.tsx

```diff
@@ -0,0 +1,16 @@
+const IShareIcon = () => {
```

Comment:
Curious why is the icon prefixed with I ?
Also I don't know if it's better to have the svg as a static file under `/public` since it doesn't have any dynamic color.